### PR TITLE
Add file based plugin configuration

### PR DESCRIPTION
**Pull request by caseydavenport**

### CODE REVIEW

File: docs/kubernetes/KubernetesIntegration.md [hunk 1]
```diff
@@ -27,36 +27,40 @@ The `calico-kubernetes` plugin integrates directly with the Kubernetes `kubelet`
 
 We recommend using the latest version of [calicoctl](https://github.com/projectcalico/calico-docker/releases) to install both `calico/node` and `calico-kubernetes` on each of your nodes.
 
-## Installing Calico on a Kubernetes Master
-The Kubernetes master does not run any pods itself, and so does not typically require the `calico-kubernetes` plugin.  It does require the `calico/node` docker container so that individual pod IPs can be learned over BGP.
+## Installing Calico Componenets
+#### 1. Install Calico
+Each Kubernetes node requires both the `calico/node` container as well as the `calico-kubernetes` plugin.  Kubernetes Masters do not need the `calico-kubernetes` plugin installed, but may do so if pods will be scheduled on the master.
```
Comment (TrimBiggs) [inline docs/kubernetes/KubernetesIntegration.md]: I assume master does not need the `calico/node` container

File: docs/kubernetes/KubernetesIntegration.md [hunk 2]
```diff
@@ -65,10 +69,6 @@ The `kubelet` can be configured to use Calico by starting it with the `--network
 On CoreOS machines, the Calico network plugin is installed to the `/etc/kubelet-plugins` directory, and so an additional flag is necessary to inform the `kubelet` of the location of the plugin.  On CoreOS, include the `--network-plugin-dir=/etc/kubelet-plugins` option when starting the `kubelet`.
 > Note: the `--network-plugin-dir=` option is only available in Kubernetes >= v1.1.0
```
Comment (TrimBiggs) [inline docs/kubernetes/KubernetesIntegration.md]: Would it be worth having the command to run here also?

**Comment by TrimBiggs** [comment]
Minor comments, LGTM

File: docs/kubernetes/KubernetesIntegration.md (hunk 1 above)
Comment (caseydavenport) [inline docs/kubernetes/KubernetesIntegration.md]: They do need it - that's why I didn't say they don't :)

File: docs/kubernetes/KubernetesIntegration.md (hunk 2 above)
Comment (caseydavenport) [inline docs/kubernetes/KubernetesIntegration.md]: The command to run will vary based on deployment.  Calico only cares that this one option is included.

File: docs/kubernetes/KubernetesIntegration.md (hunk 1 above)
Comment (TrimBiggs) [inline docs/kubernetes/KubernetesIntegration.md]: You don't have a command that explains how to install just `calico/node` on the master. I think users would immediately run the full command with the `--kubernetes` flag if you don't specify what to call for the master.  I'm not sure how much impact that would have, if any.

**Pull request merged**